src/model/message/PersistentHarvestingDelegationMessage.ts

```diff
@@ -62,7 +63,7 @@ export class PersistentHarvestingDelegationMessage extends Message {
     /**
      *
      * @param encryptMessage - Encrypted message to be decrypted
-     * @param privateKey - Recipient private key
+     * @param privateKey - Node certificate private key
      * @return {string}
      */
     public static decrypt(encryptMessage: PersistentHarvestingDelegationMessage, privateKey: string): string {
```

Comment:
This method should do the split and return a pair of signingPrivateKey and vrfPrivateKey ready to be consumed by the user (sender sends 2 keys, receipient decode 2 keys).

Reply:
I was thinking to remove it since normal user cannot decrypt message since they need the node private key. but it can be used by node owner

Reply:
I'll keep it. It's handy for at least unit testing